llvm/lib/Target/AMDGPU/VOP1Instructions.td

```diff
@@ -599,8 +599,8 @@ class Cvt_F32_F8_Pat<SDPatternOperator node, int index,
     (inst_sdwa 0, $src, 0, 0, index)
 >;
 
-let SubtargetPredicate = isGFX9Only in {
-let OtherPredicates = [HasCvtFP8VOP1Bug] in {
+let SubtargetPredicate = HasFP8ConversionInsts in {
```

Comment:
This will be true on gfx12, but it does not have these instructions. It uses opsel versions in the patterns below.

Reply:
Hence the OtherPredicates below

Reply:
But is the bug really present on gfx12?

Reply:
> But is the bug really present on gfx12?

No, the bug is that instruction requires SWDA even for the first byte. GFX12 does not have SDWA and hence does not have the bug.